sanic/worker/loader.py

```diff
@@ -55,21 +63,29 @@ def load(self) -> SanicApp:
             from sanic.app import Sanic
             from sanic.simple import create_simple_server
 
-            if self.as_simple:
-                path = Path(self.module_input)
-                app = create_simple_server(path)
+            maybe_path = Path(self.module_input)
+            if self.as_simple or (
+                maybe_path.is_dir() and os.sep in self.module_input
```

Comment:
This still only handles one kind of separator on Windows. It should be safe to check for forward and backward slashes here regardless of OS because a module name cannot contain either of those anyway.

Also, possibly would be cleaner to do this either in load or init, but not split/duplicated between the two functions as it is now.

Reply:
It is doing two different things in those locations.

Reply:
I am going to get this in now and we can refactor the whole pattern later for cleanliness later.